client/src/components/doctrack/DoctrackAngry.svelte

```diff
@@ -0,0 +1,11 @@
+<script>
+    const doctrackAngry = new URL('../../assets/logo/doctrack-angry.svg', import.meta.url);
+</script>
+
+<img src="{doctrackAngry.pathname}" alt="Doctrack angry logo" />

```

Comment:
This should be its own component in `components/icons/` instead (without a `<style>` tag for maximal flexibility).